README.md

```diff
@@ -188,6 +224,14 @@ of common shell operations (cd, cwd, pushd, popd, setenv, getenv, pushenv, popen
 arbitrary commands.
 
 ## Developing on builder
+
+### Debugging
+When debugging builder locally, use whatever python debugger you wish. You can also feed it the following command line arguments to ease the
+debugging experience:
+* --skip-install - don't install packages, assume they're already there
+* --build-dir=/path/to/other/git/repo - will jump to this directory before starting execution, helpful for debugging another project's
```

Comment:
I thought it was `--project` ?

Reply:
--project can take a path, but --build-dir will look in that directory for additional config/scripts as well (it's really only useful for setting the cwd during development, normally this app would get run from a source directory, or in an empty directory and pull its own source)